pkg/controller/applicationlayer/applicationlayer_controller.go

```diff
@@ -359,6 +362,10 @@ func updateApplicationLayerWithDefaults(al *operatorv1.ApplicationLayer) {
 	if al.Spec.ApplicationLayerPolicy == nil {
 		al.Spec.ApplicationLayerPolicy = &defaultApplicationLayerPolicyStatusType
 	}
+
+	if al.Spec.EnvoySettings == nil {
+		al.Spec.EnvoySettings = &operatorv1.EnvoySettings{}
```

Comment:
The design document suggested there were defaults for the new fields. We should set those defaults here so if in the future we decide to change them we won't change behavior for existing installs.